splunk_add_on_ucc_framework/ucc_ui_lib/src/main/resources/splunk/appserver/templates/configuration.html

```diff
@@ -0,0 +1,31 @@
+<%!
+app_name = cherrypy.request.path_info.split('/')[3]
+
+%>\
+<!doctype html>
+<html class="no-js" lang="">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>${_('Configuration')}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+</head>
+
+<body>
+    <script src="${make_url('/config?autoload=1')}" crossorigin="use-credentials"></script>
+    <script src="${make_url('/static/js/i18n.js')}"></script>
+    <script src="${make_url('/i18ncatalog?autoload=1')}"></script>
+    <script>
+        __splunkd_partials__ = ${json_decode(splunkd)};
+    </script>
+
+    <%
+    page_path = "/static/app/" + app_name + "/js/build/" + page + ".js"
```

Comment:
Just wondering, From where the value of `page ` variable is coming from?

Reply:
It seems to be coming from template engine in the backend.